server/mainPr/src/test/java/com/team23/mainPr/MainPrApplicationTests.java

```diff
@@ -29,34 +31,30 @@ class MainPrApplicationTests {
     void contextLoads() {
     }
 
-    /*
-     * refactor : 유스케이스 늘리기
-     *
-     * */
 
+    @DisplayName(value = "회원가입 입력 데이터 유효성 검증 기능 테스트 = 성공/실패 검증")
     @Test
     void validateCreateUserData() {
 
-        ResponseEntity result = memberController.checkInput(createMemberDto);
-        CommonDto dto = (CommonDto) result.getBody();
-
-        Assert.isTrue(dto.getMsg().equals("true"), "user ID, password check function test");
+        ResponseEntity<ChildCommonDto> result = memberController.checkInput(createMemberDto);
+        ChildCommonDto dto = (ChildCommonDto) result.getBody();
+        assertThat(dto.getMsg()).as("check user ID, password").isEqualTo(TRUE.getMsg());
     }
-	/*
-    *ResponseEntity 내부에 제네릭으로 body가 선언되어 있다. 가져와서 내부 데이터를 검증할수 있다.
+	/**
+    *ResponseEntity<ChildCommonDto> 내부에 제네릭으로 body가 선언되어 있다. 가져와서 내부 데이터를 검증할수 있다.
     */
 
+    @DisplayName(value = "회원가입 기능 테스트 = 응답의 아이디,비밀번호, 닉네임 검증")
     @Test
     void createUserTest() {
 
-        ResponseEntity result = memberController.createMember(createMemberDto);
-        CommonDto Cdto = (CommonDto) result.getBody();
+        ResponseEntity<ChildCommonDto> result = memberController.createMember(createMemberDto);
+        ChildCommonDto Cdto = (ChildCommonDto) result.getBody();
         MemberResponse dto = (MemberResponse) Cdto.getDto();
 
-
-        Assert.isTrue(dto.getLoginId().equals(loginId), "user ID check");
-        Assert.isTrue(dto.getPassword().equals(password), "password check");
-        Assert.isTrue(dto.getNickname().equals(nickname), "Nickname check");
+        assertThat(dto.getLoginId()).as("check user ID").isEqualTo(loginId);
+        assertThat(dto.getPassword()).as("check user password").isEqualTo(password);
+        assertThat(dto.getNickname()).as("check user Nickname").isEqualTo(nickname);
```

Comment:
AssertJ 적용 완료!

Reply:
```suggestion
        assertThat(dto.getLoginId()).isEqualTo(loginId)
                .as("check user ID");
```

`nit` 
개인적으로 이런 형태를 선호합니다.
단언문끼리 모으고, 설명은 별도 처리.